router/pump.go

```diff
@@ -214,9 +209,16 @@ func (p *LogsPump) pumpLogs(event *docker.APIEvents, backlog bool, inactivityTim
 		debug("pump.pumpLogs():", id, "pump exists")
 		return
 	}
+
+	// RawTerminal with contianer Tty=false injects binary headers into
```

Comment:
nit: type `contianer`

Reply:
I'm not sure what you're asking for here?

Reply:
whoops I typo'd the word typo.

`container` is misspelled as  `contianer` in the comment